config/targets.json

```diff
@@ -5,7 +5,8 @@
       {
         "url": "https://openopps-staging.18f.gov/"
       }
-    ]
+    ],
+    "slack_channel": "openopps-platform"
```

Comment:
Should I take out the changes to `targets.json` then, and we can add them selectively? I totally get the wanting to be careful about annoying our teammates, but not sure how to get feedback without exposing them to it. Maybe we reach out to the projects again and ask for volunteers to opt in?